CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required (VERSION 2.8)
+
+
+include(cmake/LookUp-GreatCMakeCookOff.cmake)
+subdirs(gp_emulator/gpu)
+set(LIBRARY_OUTPUT_PATH ${CMAKE_SOURCE_DIR}/build/lib)
+set(EXECUTABLE_OUTPUT_PATH ${CMAKE_SOURCE_DIR}/tests)
+string(ASCII 27 Esc)
+
+set(ColourReset "${Esc}[m")
+set(Red  "${Esc}[1;31m")
+message("${Red}Type this line or put it in ~/.bashrc or ~/.bash_profile:\nexport PYTHONPATH=$PYTHONPATH:${LIBRARY_OUTPUT_PATH} ${ColourReset}")
```

Comment:
We should do a setup.py python setuptools file to drive CMAKE so that this is unnecessary.